### ISSUE TITLE
Improving the IMU board

### ISSUE THREAD

**Issue by jtragtenberg** [YCAMInterlab/MOTIONER#3]
The soldering pads for the HMC5883L are too small. I would make it as long outside of the chip just a little bit like the ones from MPU6050, that way it is easier to check if all the pads were correctly soldered, and it makes it possible to fix the soldering with a soldering iron.

I can improve it, how do I upload to this git my changes?

**Comment by kylemcdonald** [comment]
since you've forked the repository, once you make the changes to the file you can do this:

```
$ git add -u # this stages changes to any files already added to git
$ git commit -m "made the soldering pads for the HMC5883L bigger" # this saves any staged changes
$ git push origin master # this sends your commit to the server
```

then on github there will be a button that appears that allows you to make a pull request, and we can merge it.

**Comment by jtragtenberg** [comment]
Thanks Kyle,
Just learned how to use eagle and git to do this, just requested a pull.

**Comment by kylemcdonald** [comment]
closed by #7
